### PR TITLE
String concatenation support

### PR DESCRIPTION
**Pull request by lettenj61**
Thanks for warm advice and patience, I made parser / renderer work with concatenation, but only added 1 test. I think I should add more tests (especially for parser).

### CODE REVIEW

File: src/parser/ast.rs [hunk 1]
```diff
@@ -33,6 +33,8 @@ pub enum MathOperator {
     Div,
     /// %
     Modulo,
+    /// ~
+    Tilde,
```
Comment (Keats) [inline src/parser/ast.rs]: I don't think it belongs in the math enum

File: src/renderer/tests/basic.rs [hunk 2]
```diff
@@ -53,6 +53,19 @@ fn render_variable_block_lit_expr() {
     }
 }
 
+#[test]
+fn render_variable_block_lit_concat_expr() {
+    let inputs = vec![
+        ("{{ \"music\" ~ \" start\" }}", "music start"),
+        ("{{ (4 + 4) ~ \"88\" }}", "888"),
```
Comment (Keats) [inline src/renderer/tests/basic.rs]: that's precisely something that is an issue I think, we want that kind of concatenation to be an error

**Comment by Keats** [comment]
0.11 has shipped but i'm still interested in having this added in a minor version!

**Comment by lettenj61** [comment]
Congrats for 0.11! I'm still trying to work on it, sorry for my late...

**Comment by Keats** [comment]
心配しない
Take as much time as you need and don't hesitate to ask if you have questions

**Comment by avitex** [comment]
Friendly bump. Would love to see this in Tera 👍

**Comment by Keats** [comment]
@lettenj61 do you want me to finish it if you don't have time?

**Comment by lettenj61** [comment]
@Keats I know I already ask too much, but I'm fixing previous PR currently, so I'd be very appreciated if you allow
me to continue

**Comment by Keats** [comment]
@lettenj61 no worries, take your time

**Comment by Keats** [comment]
Closing in favour of #293

**Pull request closed without merging**